### ISSUE TITLE
After an Update to 8.1.2 my order page is not working in the backoffice and neither do checkouts at the front office

### ISSUE THREAD

**Issue by MRemeikis** [PrestaShop/PrestaShop#37175]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

I updated my Prestashop from 1.7.8.2 to 8.1.2 . PHP version of the server now is 8.1. Now i have an Order manager module by prestahero on, and when i fix this error - 
![image](https://github.com/user-attachments/assets/09f9ddc4-9bd4-45ee-beb1-7de210091f00)

i get internal server error 500 after a long loading, like a template window on white screen with text and there is no information in the debug folder nor does the usual debug window open up. If the module is off, i can open the order page, where i can see all the orders, HOWEVER i cannot go into the orders themselves and it gives me a different error.

another issue i have after the update is with the PS metrics module, when it on the main page of my dashboard seems okay

![image](https://github.com/user-attachments/assets/efafec5a-da3a-44df-9907-5d57aa5945e0)

however when i go anywhere except the PS metrics module all the small icons disappear. Interesting thing, if i have the module off - the icons disappear in the main back office page too.

![image](https://github.com/user-attachments/assets/059194c6-15fe-4d6a-949e-6e9075402fbb)

and the third issue after the update that i have  is that when i try to enter checkout i get this error

![image](https://github.com/user-attachments/assets/157703ed-1c41-4893-a04f-1df4c7e82642)

### Steps to reproduce

for the first issue - 
1. Click on orders page in the back office

for the second issue - Anywhere in the back office

for the last issue
1. Add a product to cart
2. Go to cart
3. proceed to checkout

### PrestaShop version(s) where the bug happened

8.1.2

### PHP version(s) where the bug happened

8.1.29

### If your bug is related to a module, specify its name and its version

ps_metrics

**Comment by florine2623** [comment]
Hello @MRemeikis  ,

Did you use the module 1-click upgrade/autoupgrade to upgrade your shop ?

Waiting for your feedback.
Thanks!

**Comment by MRemeikis** [comment]
yes indeed I did use 1 click upgrade. Since not all my modules support the newest version of Prestashop I downloaded the 8.1.2 version from official git and used the local archive method to update Prestashop

**Comment by MRemeikis** [comment]
are there any news about the checkout error, it is pretty crucial and one of the most important features of an eshop, because i'm trying to fix it right now and cannot seem to find any information on what to do, as for logs - 
[dev-2024-10-24.log](https://github.com/user-attachments/files/17503669/dev-2024-10-24.log)

**Comment by MRemeikis** [comment]
any help with the checkouts that are not working? (the last thing i added in the screenshots of the original issue)

**Comment by MRemeikis** [comment]
i also opened a topic in official prestashop forums https://www.prestashop.com/forums/topic/1091933-after-upgrading-to-version-812-my-checkout-does-not-work/

**Comment by AureRita** [comment]
Hi @MRemeikis 

Thank you for your report, currently I tried to reproduce all your issue but I can't reproduce it as you can see : 

https://github.com/user-attachments/assets/bb288aa8-7240-4326-87a1-ed2c336d57b1

I can help you for your issue on the icons, it's a cache effect, you can do a hard refresh of your webpage with "CTRL+SHIFT+R" 

When you'll see the video, did I miss something to reproduce your issue ? 

Waiting for feedback

**Comment by MRemeikis** [comment]
Hello! So the icon issue i fixed a long time ago, however the checkout issue with the cookies like in the video, did not solve the issue. Could you replicate the issue with the checkout while having dev mode on, so that we could check if it is the same error? For me it sais that the dispatcher is trying to reach the OrderController and cannot find it however when i did the logging, with all acceccable controllers for the front i had this ![Image](https://github.com/user-attachments/assets/ddb371fe-4f8b-4b74-9010-03188d4d5dbf)
which means, the controller is found with the dispatcher when it scans through during the call, however when it needs to be loaded up it just gets missed
